src/config_trapdoor.c

```diff
@@ -853,10 +848,9 @@ TbBool load_trapdoor_config(const char *conf_fname, unsigned short flags)
 {
     static const char config_global_textname[] = "global traps and doors config";
     static const char config_campgn_textname[] = "campaign traps and doors config";
-    char *fname;
-    TbBool result;
-    fname = prepare_file_path(FGrp_FxData,conf_fname);
-    result = load_trapdoor_config_file(config_global_textname,fname,flags);
+
```

Comment:
@Trass3r are all these empty lines intended? I highlight just one, but happens often.

Reply:
Not really. It does run clang-format on the patches.

Reply:
Sorry, I don't know what that means.

Reply:
It's a tool to automatically format code.